components/js/SlickGrid.ts

```diff
@@ -343,15 +379,16 @@ export class SlickGrid implements OnChanges, OnInit, OnDestroy, AfterViewInit {
 
     /* tslint:disable:member-ordering */
     private getColumnEditor = (column: any): any => {
+        if (this.isColumnEditable && !this.isColumnEditable(this.getColumnIndex(column))) {
+            return undefined;
+        }
+
         let columnId = column.id;
-        let isEditable = this.editableColumnIds && this.editableColumnIds.indexOf(columnId) !== -1;
         let isColumnLoading = this.columnsLoading && this.columnsLoading.indexOf(columnId) !== -1;
-        if (isEditable) {
-            return isColumnLoading
-                ? getDisabledEditorClass('')
```

Comment:
If this function returns truthy, [slickgrid will treat the cell as editable](https://github.com/mleibman/SlickGrid/blob/master/slick.grid.js#L2546). This turns the cell UI contents invisible (as if the cell were in the process of being edited) when a cell with `getDisabledEditorClass` is clicked, which is not ideal.